packages/cli-repl/src/format-output.ts

```diff
@@ -119,9 +119,25 @@ function formatSimpleType(output: any, options: FormatOptions): any {
 }
 
 function formatCollections(output: CollectionNamesWithTypes[], options: FormatOptions): string {
-  const tableEntries = output.map(
-    (coll) => [clr(coll.name, 'bold', options), coll.badge]
-  );
+  const systemCollections: CollectionNamesWithTypes[] = [];
+  const otherCollections: CollectionNamesWithTypes[] = [];
+
+  output.forEach(coll => {
+    if (coll.name.startsWith('system.')) {
+      systemCollections.push(coll);
+    } else {
+      otherCollections.push(coll);
+    }
+  });
+
+  const tableEntries = [
+    ...otherCollections.map(
+      coll => [clr(coll.name, 'bold', options), coll.badge]
+    ),
+    ...systemCollections.map(
+      coll => [`\u001b[2m${clr(coll.name, 'bold', options)}`, coll.badge]
```

Comment:
It’s too bad that ansi-escape-sequences doesn’t support this out of the box – but in any case, we want to only apply the `\u001b[2m` here if `options.colors` is set, I think

Reply:
Yeah I took a look but unfortunately they don't :/ But totally missed the options.color thing...